radio/src/gui/colorlcd/view_statistics.cpp

```diff
@@ -240,26 +241,26 @@ void DebugViewPage::build(FormWindow* window)
   FlexGridLayout grid(dbg_col_dsc, row_dsc, 0);
 
   auto line = form->newLine(&grid);
-  line->padAll(4);
+  line->padAll(2);
 
   // Mixer data
   new StaticText(line, rect_t{}, STR_TMIXMAXMS, 0, COLOR_THEME_PRIMARY1);
   new DynamicNumber<uint16_t>(
       line, rect_t{}, [] { return DURATION_MS_PREC2(maxMixerDuration); },
-      PREC2 | COLOR_THEME_PRIMARY1, nullptr, "ms");
+      PREC2 | COLOR_THEME_PRIMARY1, nullptr, STR_MS);
 
   line = form->newLine(&grid);
-  line->padAll(4);
+  line->padAll(2);
 
   // Free mem
   new StaticText(line, rect_t{}, STR_FREE_MEM_LABEL, 0, COLOR_THEME_PRIMARY1);
   new DynamicNumber<int32_t>(
-      line, rect_t{}, [] { return availableMemory(); }, COLOR_THEME_PRIMARY1,
-      nullptr, "b");
+      line, rect_t{}, [] { return availableMemory(); }, COLOR_THEME_PRIMARY1, 
+      nullptr, STR_BYTES);
```

Comment:
@philmoz Any idea on how to insert a space before the suffix (STR_BYTES and the above STR_MS) ... as it would just look a little nicer.

Reply:
There are two places where STR_BYTES are used on B&W radios at the moment.
One option would be to add the space to all the TR_BYTES translations then change the two STR_BYTES references to STR_BYTES+1.

Reply:
Or add a static char[] to the view_statistics.cpp code and generate the string with space on page load.